### PR TITLE
easier book crafting recipe

### PR DESCRIPTION
**Pull request by c-sig**
leather is in really short demand due to no other way of obtaining it outside of custom mobs and there is no real reason for books to be that expensive to make. this leaves more chances for leather to be used for more custom stuff than wasting it on bookshelves

### CODE REVIEW

**Comment by ronnor88** [comment]
the paper is a must but what item do you recommend to add instead of the leather i recommend things like wood or string(your idea) maybe even iron nuggets

**Comment by c-sig** [comment]
my brother in christ this is a pull request to add the string + paper book recipe into the survival server not a feature request i dont plan on suggesting any other alternatives

**Comment by ronnor88** [comment]
k

File: servers/minecraft/plugins/Geary/mineinabyss/items/misc/book.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      items:
+        S:
+          prefab: mineinabyss:kuongatari_silk
+        P:  
+          type: PAPER
+      configuration: |-
+        |SP |
+        |SP |
+        |SP |
+           - !<shaped>
```
Comment (Boy0000) [inline servers/minecraft/plugins/Geary/mineinabyss/items/misc/book.yml:26]: shaped tag should not be here

File: servers/minecraft/plugins/Geary/mineinabyss/items/misc/book.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      items:
+        S:
+          type: STRING
+        P:  
+          type: PAPER
+      configuration: |-
+        |SP |
+        |SP |
+        |SP |
+
```
Comment (Boy0000) [inline servers/minecraft/plugins/Geary/mineinabyss/items/misc/book.yml:36]: point of this when above covers all string?

File: servers/minecraft/plugins/Geary/mineinabyss/items/misc/book.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          prefab: mineinabyss:silkfang_silk
+        P:  
+          type: PAPER
+      configuration: |-
+        |SP |
+        |SP |
+        |SP |
+           - !<shaped>
+      items:
+        S:
```
Comment (Boy0000) [inline servers/minecraft/plugins/Geary/mineinabyss/items/misc/book.yml:18]: indent error again

File: servers/minecraft/plugins/Geary/mineinabyss/items/misc/book.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
+- !<looty:recipes>
+  discoverRecipes: true
+  result:
+    type: BOOK
+  recipes:
+    - !<shaped>
```
Comment (Boy0000) [inline servers/minecraft/plugins/Geary/mineinabyss/items/misc/book.yml:6]: add a group here, preferably the vanilla book one to bundle it with vanilla recipe in recipe book

**Review by Boy0000 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**